src/index.html

```diff
@@ -17,6 +29,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Logo"
+        />
+      </a>
+
+      <nav>
+        <ul>
+          <li>
+            <a
+              href="#"
+              class="is-active aa"
+            >
+              Apple
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="aa"
+            >
+              Samsung
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="aa"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              data-qa="hover"
+              class="aa"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="aa"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
```

Comment:
The `href` attribute is set to `#`, which is a placeholder. Consider updating it to point to the actual URL or route for the 'Tablets' link.